docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Commencing 
+FROM ubuntu:18.04
+
+#ENV DEBIAN_FRONTEND=noninteractive
+#ENV TZ=US/Chicago
+#RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+## Updates and software that I like
+# Install vnc, xvfb in order to create a 'fake' display and firefox
+RUN apt-get update
+RUN apt-get install -y htop tmux
+RUN apt-get install -y vim=2:8.0.1453-1ubuntu1
+RUN apt-get install -y neovim
+RUN apt-get update
+RUN apt-get install -y ssh 
+RUN apt-get install -y curl
+RUN apt-get install -y python 
```

Comment:
i think we don't need this (this installs python 2)

Reply:
correct